data-plane/config/source/500-dispatcher.yaml

```diff
@@ -23,6 +23,7 @@ metadata:
     app: kafka-source-dispatcher
     kafka.eventing.knative.dev/release: devel
 spec:
+  replicas: 0
```

Comment:
This won't work with the downstream operator or any GitOps setup because these systems reconcile these manifests so they will revert any changes done by the autoscaler.

Reply:
Oh boy - yeah, the operator world. Thanks for catching that!